packages/go/index.ts

```diff
@@ -501,17 +520,54 @@ export async function build({
     throw error;
   } finally {
     try {
-      await cleanupFileSystem(undoFileActions, undoDirectoryCreation);
+      await cleanupFileSystem(
+        undoFileActions,
+        undoDirectoryCreation,
+        undoFunctionRenames
+      );
     } catch (error) {
       console.log(`Build succeeded, but cleanup failed: ${error.message}`);
       debug('Cleanup Error: ' + error);
     }
   }
 }
 
+async function renameHandlerFunction(fsPath: string, from: string, to: string) {
+  let fileContents = (await readFile(fsPath)).toString('utf-8');
+
+  const fromRegex = new RegExp(`\\b${from}\\b`, 'g');
```

Comment:
Relying on word boundaries here accounts for some things, like replacing all instances of a function name, like:

```go
var newIndex = Index

func Index(...)
```

But this will also catch the same value in comments and strings. I've asked internally about a better way to approach this, likely with actual AST parsing.

@TooTallNate and I agree that this is shippable as is, but we'd still work on the more complete solution immediately.